markets/perps-market/test/integration/Liquidation/Liquidation.margin.test.ts

```diff
@@ -208,14 +214,84 @@ describe('Liquidation - margin', async () => {
       assertBn.equal(await systems().PerpsMarket.getAvailableMargin(2), bn(16_550));
     });
   });
+  describe('price change - available margin 0 ', () => {
+    [
+      bn(31000), // btc
+      bn(1775), // eth
+      bn(3), // link
+      bn(1), // arb
+      bn(1.13), // op
+    ].forEach((price, i) => {
+      before(`change ${perpsMarketConfigs[i].token} price`, async () => {
+        await perpsMarkets()[i].aggregator().mockSetCurrentPrice(price);
+      });
+    });
+
+    [
+      bn(-1150), // btc
+      bn(-4900), // eth
+      bn(-4100), // link
+      bn(-5250), // arb
+      bn(-4600), // op
+    ].forEach((pnl, i) => {
+      it(`should have correct position pnl for ${perpsMarketConfigs[i].token}`, async () => {
+        const [positionPnl] = await systems().PerpsMarket.getOpenPosition(
+          2,
+          perpsMarkets()[i].marketId()
+        );
+        assertBn.equal(positionPnl, pnl);
+      });
+    });
+    it('has correct available margin', async () => {
+      assertBn.equal(await systems().PerpsMarket.getAvailableMargin(2), bn(0));
+    });
+  });
+
+  describe('minimumPositionMargin increased -> eligible for liquidation', () => {
+    const restoreMinimumPositionMargin = snapshotCheckpoint(provider);
+    before('set minimumPositionMargin for OP to 50', async () => {
+      const opMarketId = perpsMarkets()[4].marketId();
+      const initialMarginFraction = bn(1.5);
+      const maintenanceMarginFraction = bn(0.75);
+      const maxLiquidationLimitAccumulationMultiplier = bn(1);
+      const liquidationRewardRatio = bn(0.05);
+      const maxSecondsInLiquidationWindow = bn(10);
+      const minimumPositionMargin = bn(50); // this is the only change from the initial values
+      await systems()
+        .PerpsMarket.connect(marketOwner())
+        .setLiquidationParameters(
+          opMarketId,
+          initialMarginFraction,
+          maintenanceMarginFraction,
+          maxLiquidationLimitAccumulationMultiplier,
+          liquidationRewardRatio,
+          maxSecondsInLiquidationWindow,
+          minimumPositionMargin
+        );
+    });
+    // Changing minimumPositionMargin does not have an affect on available margin
+    it('has correct available margin', async () => {
+      assertBn.equal(await systems().PerpsMarket.getAvailableMargin(2), bn(0));
```

Comment:
We have this `getAvailableMargin`, that just looks at the total account collateral value + the account PNL.
So it's missing the liquidations considerations, eg most of the time I will not be able to withdraw what it returns.. 
Might be worth calling this something else, and make `getAvailableMargin` take liquidations concerns into account?

Reply:
the `getAvailableMargin` is what gets checked against your maintenance margin for liquidations.  its not meant to be a value that is withdrawable.  its something to display on the UI to check your margin against your pnls.  maybe a `getWithdrawableAmount` which subtracts out the maintenace margin could be nice to add

Reply:
> the `getAvailableMargin` is what gets checked against your maintenance margin for liquidations. its not meant to be a value that is withdrawable. its something to display on the UI to check your margin against your pnls. maybe a `getWithdrawableAmount` which subtracts out the maintenace margin could be nice to add

Yea that what I want. And I like that name too!